### ISSUE TITLE
Barcode print wish list

### ISSUE THREAD

**Issue by odiea** [jekkos/opensourcepos#290]
In our store we sell items separately. It would be a nice added feature if there was a repeat feature built into barcode printing. When we enter an item with a qty of 5 we would use 5 printed barcode labels.  Something to think about in a future release.

**Comment by odiea** [comment]
It would also be a good idea to Include barcode printing in receivings. We could never use this fine program because a person inputting in items would be lost trying to find and print barcodes from the items list. Our last report showed we had over 250000 items. Getting lost would be easy. The date function was nice but I noticed it would also include items that were sold that same day.

**Comment by daN4cat** [comment]
I see you have two requirements there, one is for repeat barcode printing, the other to select items by date range just created not sold.

**Comment by odiea** [comment]
The above requests could be eliminated if Receivings had a barcode feature.

**Comment by odiea** [comment]
I fixed my issue by adding another checkbox field to items. Now I can filter the items by the new field and also bulk edit the items.

**Comment by jekkos** [comment]
Cool. well done. how do you keep track of your changes here? Once you upgrade to a later version you lose most of the things you did ? Might be easier to use git (and make some PR's for these kind of features)?

**Comment by odiea** [comment]
Yes I do use git. Most of the time it does not screw up my changes but I do back up my sites before I try a pull from your branch..

**Comment by odiea** [comment]
http://ospos.25u.com:8080/tabbed-dev/index.php/sales. This has my current updates. No edit in Takings, Items filter for barcode (bulk update removes the tax percent if not manually added), Updated low inventory based on reorder level and full customer information.

**Comment by daN4cat** [comment]
The filtering of Items could be done against transactions that are not sales or receivings, it's just an extra check in the model against the transactions table

**Comment by odiea** [comment]
@daN4cat  My logic behind doing it this way is The employee would 3 to 20 items at a time then print the barcodes. This way if a batch doesn't get printed they can easily reprint them. Then the Table can be cleared with the bulk update for the next batch of items.

**Comment by daN4cat** [comment]
Sounds, I probably misunderstood the use case.

**Comment by odiea** [comment]
But the repeat function would be nice. We get multiple items and sell them individually. If we enter 6 items with the same item id then 6 barcodes would need to be printed.

**Comment by naxyr** [comment]
@odiea any update on barcode repeat function.
